src/main.py

```diff
@@ -1,18 +1,21 @@
 import os
-import requests
 import pandas as pd
+from dune_client.client import DuneClient
+from dune_client.query import QueryBase
 from sqlalchemy import create_engine
 from dotenv import load_dotenv
 
 from src.types import DUNE_TO_PG
 
 load_dotenv()
-DUNE_API_KEY = os.getenv("DUNE_API_KEY")
-DB_URL = os.getenv("DB_URL")
-TABLE_NAME = "dune_data"
+DUNE_API_KEY = os.environ.get("DUNE_API_KEY")
+DB_URL = os.environ.get("DB_URL")
 
-API_URL = "https://api.dune.com/api/v1/query/4132129/results"
-HEADERS = {"x-dune-api-key": DUNE_API_KEY}
+# TODO(bh2smith): parse config file for most of the following stuff
```

Comment:
Introduce Yaml or Toml Configuration

The only environment variables should be secrets (i.e. `DB_URL` and `DUNE_API_KEY`). All other runtime arguments should be passed to the program via a Config structure that is parsed from a file.

Reply:
Minor preference for yaml here. It's more convoluted and has more pitfalls, but TOML seems kinda limiting at first glance.

Reply:
https://github.com/bh2smith/dune-sync/issues/4

Reply:
The one nice benefit I see for toml. Is that python offers native support without having to depend on a third party library.